src/components/bottom_section/Buttons.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+
+import {
+  BLANK,
+  DIVISION,
+  MAX_CHARACTERS,
+  MINUS,
+  MULTIPLY,
+  PERCENTAGE,
+  PLUS,
+} from '../../constants/Constants';
+import {
+  AllClearButton,
+  CommaButton,
+  DivisionButton,
+  EqualsButton,
+  MinusButton,
+  MultiplyButton,
+  PercentageButton,
+  PlusButton,
+  SignButton,
+} from './buttons/index';
+
+const Buttons = ({
+  setDisplayValue,
+  displayValue,
+  waitingValue,
+  setWaitingValue,
+  setCurrentValue,
+  currentValue,
+  operation,
+  setOperation,
+}) => {
+  const currentValueAsNumber = Number(currentValue);
```

Comment:
Can't we keep `currentValue` as number always? Why does it become not number type?